tests/ui/imports/redundant-import-issue-121915.rs

```diff
@@ -0,0 +1,12 @@
+//@ revisions: edition2018 edition2021
+//@ [edition2018] edition:2018
+//@ [edition2021] edition:2021
```

Comment:
Any specific reason to test this on both editions?
2018 and 2021 are identical with regards to name resolution (unlike 2015).